consensus/src/tests/mock_execution_controller.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2021 MASSA LABS <info@massa.net>
+
+use execution::{ExecutionCommand, ExecutionCommandSender};
+use models::{Block, BlockHashMap};
+use time::UTime;
+use tokio::{
+    sync::mpsc::{channel, Receiver, Sender},
+    time::sleep,
+};
+
```

Comment:
I think you want to add something similar to the current `PoolCommandSink`, to prevent deadlock if the channel were to fill-up, and to properly shut it down at the end of the test.

Reply:
Looks good otherwise!